Gopkg.toml

```diff
@@ -68,3 +69,7 @@ ignored = [
 [[constraint]]
   name = "github.com/jteeuwen/go-bindata"
   revision = "v3.0.7"
+
+[[constraint]]
+  name = "github.com/gojuno/minimock"
+  branch = "master"
```

Comment:
Don't relay on the latest master. Add revision.

Reply:
ok